### PR TITLE
Update RU strings & general fixes

### PR DESCRIPTION
**Pull request by Displax**

### CODE REVIEW

File: stub/src/main/res/values-ru/strings.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <resources>
-    <string name="upgrade_msg">Обновите Magisk Manager для завершения установки. Продолжить?</string>
-    <string name="no_internet_msg">Пожалуйста, подключитесь к интернету! Требуется обновление Magisk Manager.</string>
+    <string name="upgrade_msg">Обновите приложение Magisk для завершения установки. Продолжить?</string>
+    <string name="no_internet_msg">Пожалуйста, подключитесь к интернету! Требуется обновление приложения Magisk.</string>
```
Comment (yurikoles) [inline stub/src/main/res/values-ru/strings.xml:4]: `Magisk Manager` is the proper noun of the app, but you changed it to `Magisk application`, which loses sense.

@Displax, аналогия: ОС Окна от Мелкомягких.

Comment (yurikoles) [reply stub/src/main/res/values-ru/strings.xml:4]: МежДелМаш поглотил Красную Шапочку :)

**Review by yurikoles — CHANGES_REQUESTED** [review]

**Comment by Displax** [comment]
@yurikoles 
Менеджера больше нет.
https://github.com/topjohnwu/Magisk/blob/master/docs/releases/22000.md#magisk-manager-is-deadlong-live-the-magisk-app
Упоминания слова "manager" удалено отовсюду и заменено на "app".
https://github.com/topjohnwu/Magisk/commit/b9669f54f739d3d72c579f3f6847965f4bd5119d
https://github.com/topjohnwu/Magisk/commit/6951d926f7affde1cd7965be6e5ce6091f90c798
https://github.com/topjohnwu/Magisk/commit/75584e2b198b5595c748a73750e2e1d6dbfc629e#diff-5e01f7d37a66e4ca03deefc205d8e7008661cdd0284a05aaba1858e6b7bf9103

**Comment by topjohnwu** [comment]
@Displax may you please remove this commit 843dd4bdd4ff3acaf2ae2d99901f4c1c86006ab3? Thank you

**Comment by Displax** [comment]
@topjohnwu Done!

**Review by topjohnwu — APPROVED** [review]

**Pull request merged**